libdirections-offboard/src/test/java/com/mapbox/navigation/route/offboard/router/NavigationRouteTest.kt

```diff
@@ -52,7 +52,7 @@ class NavigationRouteTest {
     @Test
     @Throws(Exception::class)
     fun sanityTest() {
-        val navigationRoute = NavigationRoute.builder(context)
+        val navigationRoute = NavigationOffboardRoute.builder(ACESS_TOKEN, context)
```

Comment:
```suggestion
        val navigationRoute = NavigationOffboardRoute.builder(ACESS_TOKEN, context)
```
many tests have the same line. maybe move it to a separate fun. 
like `buildNavigationRoute() = NavigationOffboardRoute.builder(ACESS_TOKEN, context)`